qa/rpc-tests/signrawtransaction_offline.py

```diff
@@ -35,18 +36,11 @@ def run_test(self):
         sign_inputs = [{'txid': txid, 'vout': 0, 'scriptPubKey': scriptpubkey, 'amount': 10}]
 
         create_hex = self.nodes[0].createrawtransaction(create_inputs, {taddr: 9.9999})
-        print "create:"
-        print create_hex
 
         signed_tx = offline_node.signrawtransaction(create_hex, sign_inputs, privkeys)
-        print "sign:"
-        print signed_tx
-
-        signed_hex = signed_tx['hex']
-        #print "decoded:"
-        #print self.nodes[0].decoderawtransaction(signed_hex)
-        print "sent:"
-        print self.nodes[0].sendrawtransaction(signed_hex)
+
+        # If we return the transaction hash, then we have have not thrown an error (success)
+        assert_true(len(self.nodes[0].sendrawtransaction(signed_tx['hex'])) > 0)
```

Comment:
Refactor this change into the previous commit, so that the test was originally failing.

Also, if you call `offline_node.sendrawtransaction(...)` first, then you should be able to assert that `nodes[0].sendrawtransaction(...)` returns the same result.

Reply:
I think that it was failing when we tried to print below, but I will do this to make it more clear.

Reply:
Calling `offline_node.sendrawtransaction(...)` fails because `JSONRPC error: 16: tx-overwinter-not-active`